db/all_in_one_create_jewlz.sql

```diff
@@ -175,7 +175,7 @@ CREATE TABLE `product` (
on\'t let go, Jack.','Silver','Necklace','Women','silver_pendant_necklace.png','20 inches',100,60,150,'None.',0,0,0,0,0,0),(20058,'Pendant Necklace','I won\'t let go, Jack.','Silver','Necklace','Women','silver_pendant_necklace.png','22 inches',100,60,150,'None.',0,0,0,0,0,0),(20059,'Pearl Necklace','Impress your friends!','Pearl','Necklace','Women','pearl_necklace.png','16 inches',100,85,212.5,'None.',0,0,0,0,0,0),(20060,'Pearl Necklace','Impress your friends!','Pearl','Necklace','Women','pearl_necklace.png','18 inches',100,85,212.5,'None.',0,0,0,0,0,0),(20061,'Pearl Necklace','Impress your friends!','Pearl','Necklace','Women','pearl_necklace.png','20 inches',100,85,212.5,'None.',0,0,0,0,0,0),(20062,'Pearl Necklace','Impress your friends!','Pearl','Necklace','Women','pearl_necklace.png','22 inches',100,85,212.5,'None.',0,0,0,0,0,0),(20063,'Crystal Necklace','For people who like astrology.','Crystal','Necklace','Women','crystal_necklace.png','16 inches',100,100,250,'None.',0,0,0,0,0,0),(20064,'Crystal Necklace','For people who like astrology.','Crystal','Necklace','Women','crystal_necklace.png','18 inches',100,100,250,'None.',0,0,0,0,0,0),(20065,'Crystal Necklace','For people who like astrology.','Crystal','Necklace','Women','crystal_necklace.png','20 inches',100,100,250,'None.',0,0,0,0,0,0),(20066,'Crystal Necklace','For people who like astrology.','Crystal','Necklace','Women','crystal_necklace.png','22 inches',100,100,250,'None.',0,0,0,0,0,0),(20067,'Owl Pendant Necklace','This thing\'s a hoot.','Owl','Everyday','Women','owl_pendant_necklace.png','16 inches',100,22,55,'None.',0,0,0,0,0,0),(20068,'Owl Pendant Necklace','This thing\'s a hoot.','Owl','Everyday','Women','owl_pendant_necklace.png','18 inches',100,22,55,'None.',0,0,0,0,0,0),(20069,'Owl Pendant Necklace','This thing\'s a hoot.','Owl','Everyday','Women','owl_pendant_necklace.png','20 inches',100,22,55,'None.',0,0,0,0,0,0),(20070,'Owl Pendant Necklace','This thing\'s a hoot.','Owl','Everyday','Women','owl_pendant_necklace.png','22 inches',100,22,55,'None.',0,0,0,0,0,0),(20071,'Rose Stud Earings','For confused teenagers.','Rose','Everyday','Women','rose_ear_rings.png','One Size',3,10,25,'None.',0,0,0,0,0,0),(20072,'Cat Ring','It won\'t bring Tuffy back, but it\'s a good idea anyway.','Silver','Everyday','Women','cat_ring.png','5',100,11,27.5,'None.',0,0,0,0,0,0),(20073,'Cat Ring','It won\'t bring Tuffy back, but it\'s a good idea anyway.','Silver','Everyday','Women','cat_ring.png','6',100,11,27.5,'None.',0,0,0,0,0,0),(20074,'Cat Ring','It won\'t bring Tuffy back, but it\'s a good idea anyway.','Silver','Everyday','Women','cat_ring.png','7',100,11,27.5,'None.',0,0,0,0,0,0),(20075,'Cat Ring','It won\'t bring Tuffy back, but it\'s a good idea anyway.','Silver','Everyday','Women','cat_ring.png','8',100,11,27.5,'None.',0,0,0,0,0,0),(20076,'Silver Bangles','Serve chilled, with white wine and pain killers.','Silver','Everyday','Women','silver_bangle.png','One Size',0,44,110,'None.',0,0,0,0,0,0);
```

Comment:
@mferrall this line will need to be changed in all iterations in `db/`, not just the `create_all` script.